src/scenes/home/informationForm/informationForm.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import Section from 'shared/components/section/section';
+import FormButton from 'shared/components/form/formButton/formButton';
+import { patchBackend } from 'shared/utils/apiHelper';
+import Identifier from './formComponents/identifier';
+import Interests from './formComponents/interests';
+import WorkInfo from './formComponents/workInfo';
+import SchoolInfo from './formComponents/schoolInfo';
+import MilitaryInfo from './formComponents/militaryInfo';
+import styles from './informationForm.css';
+
+class SignupInformation extends Component {
+
+  constructor(props) {
+    super(props);
+    this.onIdentifierStatusChange = this.onIdentifierStatusChange.bind(this);
+    this.onCheckBoxChange = this.onCheckBoxChange.bind(this);
+    this.state = {
+      error: false,
+      isValid: true,
+      success: false,
+      interests: new Set(),
+      step: 0
+    };
+  }
+
+  onIdentifierStatusChange = (e, value) => {
+    this.setState({ [e.target.id]: value });
+  }
+
+  onCheckBoxChange = (value) => {
+    this.setState({ interests: this.state.interests.add(value) });
+  }
+
+  previousPage = () => {
+    if (this.state.step < 1) {
+      return;
+    }
+    this.setState({ step: this.state.step -= 1 });
+  }
+
+  saveAndContinue = () => {
+    patchBackend('users', {
+      user: {
+        education_level: this.state.schoolLevel,
+        scholarships: this.state.scholarships,
+        employment_status: this.state.workInfo,
+        company_name: this.state.company,
+        company_role: this.state.role,
+        volunteer: this.state.identifier,
+        branch_of_service: this.state.branch,
+        interests: [...this.state.interests]
+      }
+    });
+    this.setState({ step: this.state.step += 1 });
+  }
+
+  showStep = () => {
```

Comment:
Can you add a comment explaining how the steps work, maybe adding the step names in there as well?

1false 1true etc is a little cryptic.